server/src/main/java/org/elasticsearch/rest/BaseRestHandler.java

```diff
@@ -147,6 +150,19 @@ public void close() {
         }
     }
 
+    private boolean assertConsumesSupportedParams(@Nullable Set<String> supported, RestRequest request) {
+        if (supported != null) {
+            final var supportedAndCommon = new TreeSet<>(supported);
+            supportedAndCommon.addAll(List.of("error_trace", "filter_path", "format", "pretty", "human"));
```

Comment:
Do we have these common params defined anywhere so this isn't an arbitrary list?

Reply:
Eh not all of them. There's `org.elasticsearch.rest.BaseRestHandler#ALWAYS_SUPPORTED` but that doesn't include `error_trace`. I pushed a commit that uses `ALWAYS_SUPPORTED`. Not a huge deal here in any case, if we get this wrong then this assertion will trip.